quadratic/quadratic.go

```diff
@@ -19,22 +16,21 @@ x2 = -5 */
 func main() {
 	fmt.Println("The program finds solutions of a quadratic equation.")
 	fmt.Println("Enter the coefficients A, B and C:")
-	var A float64
-	var B float64
-	var C float64
+	var A, B, C float64
 	fmt.Scan(&A, &B, &C)
 	var D float64 = B*B - 4*A*C
-	if D > 0 {
-		var x1 float64 = (-B + math.Sqrt(D)) / (2 * A)
-		var x2 float64 = (-B - math.Sqrt(D)) / (2 * A)
-		fmt.Printf("x1: %v\n", x1)
-		fmt.Printf("x2: %v", x2)
-	}
-	if D == 0 {
-		var x float64 = -B / (2 * A)
-		fmt.Printf("x: %v", x)
+	if A == 0 {
```

Comment:
If A = 0, then the equation is linear: Bx + C = 0. Therefore, `x = 0` is incorrect. Could you please change a program to handle this case correctly?

Btw, you are implementing a program for the next assignment: https://github.com/prog-1/conditions#quadratic-equation-6-points. Once solved correctly, just copy it to `conditions` repository. You will get 6 points for it ;)